content/en/database_monitoring/setup_mysql/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Setting up MySQL
+kind: documentation
+description: Setting up Database Monitoring on a MySQL database
+disable_sidebar: true
+---
+
+{{< site-region region="us3,gov" >}}
```

Comment:
Just a heads up that there is no TOC on this page or the other setup index pages, so the site selector doesn't show on these pages. May want to move this into an alert for better visibility.